example/styles.css

```diff
@@ -1102,6 +1102,20 @@ a.cf-link--disabled {
 }
 
 /**
+ * Wizard
+ */
+
+.cf-wizard .cf-modal__footer {
+  display: flex;
```

Comment:
Does the component use flexbox or just the example?  If the component requires flexbox can we use `cf-component-flex` instead?